### ISSUE TITLE
[Feature Request] HTML Widget

### ISSUE THREAD

**Issue by dskrepps** [joelshepherd/tabliss#251]
I noticed a nice [weather widget](https://weatherwidget.io/) I wanted to add and was then surprised there is no HTML widget option. I then edited in the code via my browser's inspector to see how it would look anyway, and found the Content Security Policy blocked third party scripts, which might be needed for many third party widgets.

Here's what I would have if we had this feature:
![Screenshot_2020-05-21_19-27-01](https://user-images.githubusercontent.com/868050/82615800-55ccad00-9b99-11ea-8c7d-6d6a64702f8a.png)

**Comment by trickypr** [comment]
Do you mean an [IFrame](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) widget, so you can embed different websites into the page?

**Comment by dskrepps** [comment]
No... actual html code. See the "get code" button on the [weather widget](https://weatherwidget.io/) page as an example.

I believe it's a common feature of dashboards and homepages which utilize widgets to have an html widget to allow you to add anything.

**Comment by joelshepherd** [comment]
I could possibly add a plain HTML widget, but I would not be able to call any javascript on it's own. Web extensions are strictly forbidden to run any js that isn't part of the original extesion bundle.
